features/steps/simple.py

```diff
@@ -0,0 +1,54 @@
+from behave import *
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+use_step_matcher("re")
+
+driver = webdriver.Chrome()
+
+
+@given("I have access to the internet")
+def step_impl(context):
+    """
+    :type context: behave.runner.Context
+    """
+    driver.get("http://www.google.com")
+    assert "Google" in driver.title
+
+
+@step("I have entered d9nich in google search")
+def step_impl(context):
+    """
+    :type context: behave.runner.Context
+    """
+    elem = driver.find_element_by_name("q")
+    elem.clear()
+    elem.send_keys("d9nich")
+    elem.send_keys(Keys.RETURN)
+
+
+@step('page contains "Danylo Halaiko"')
```

Comment:
Could you make this step parametrized, so "Danylo Halaiko" could be an arbitrary text?

Reply:
fix with [scenario outline commit](https://github.com/d9nchik/selenium/pull/1/commits/bff89745c5f5efd0df528895ce0640828b35b9bc)